zerver/openapi/zulip.yaml

```diff
@@ -19653,43 +19645,15 @@ components:
         type:
           type: string
           description: |
-            The type of the onboarding step. Valid values are either
-            `"hotspot"` or `"one_time_notice"`.
+            The type of the onboarding step. Valid value is `"one_time_notice"`.
+
+            **Changes**: Removed type `"hotspot"` in Zulip 9.0 (feature level 259).
 
-            **Changes**: New in Zulip 8.0 (feature level 233).
+            New in Zulip 8.0 (feature level 233).
         name:
           type: string
           description: |
             The name of the onboarding step.
-        title:
-          type: string
-          description: |
-            The title of the onboarding step, as displayed to the user.
-
-            Only present for onboarding steps with type `"hotspot"`.
-        description:
-          type: string
-          description: |
-            The description of the onboarding step, as displayed to the
-            user.
-
-            Only present for onboarding steps with type `"hotspot"`.
-        delay:
-          type: number
-          description: |
-            The delay, in seconds, after which the user should be shown
-            the onboarding step.
-
-            Only present for onboarding steps with type `"hotspot"`.
-        has_trigger:
-          type: boolean
-          description: |
-            Identifies if the onboarding step will activate only when some
-            specific event occurs.
-
-            Only present for onboarding steps with type `"hotspot"`.
-
-            **Changes**: New in Zulip 8.0 (feature level 230).
```

Comment:
Removed this fields instead of documenting it as deprecated as we mention 
> We expect that only official Zulip clients will interact with this data.

and only web app is using it. So, its fine to just remove it.

Reply:
Yep, sounds good.